README.md

```diff
@@ -1,2 +1,42 @@
-# odh-manifests
-A repository for Open Data Hub Kustomize manifests extending upstream Kubeflow manifests
+# Open Data Hub Manifests
+A repository for [Open Data Hub](https://opendatahub.io) components Kustomize manifests.
+
+## Community
+
+* Website: https://opendatahub.io
+* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Mailing lists: https://opendatahub.io/community.html
+* Community meetings: https://gitlab.com/opendatahub/opendatahub-community
+
+## Components
+
+Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers. The components currently available are:
+
+* [JupyterHub](jupyterhub/README.md)
+* Spark Operator
```

Comment:
@vpavlin the link for JupyterHub is broken. What about the other components, we don't have readmes

Reply:
It is not broken, but since it is not part of the PR it will only start working when the PR is merged.

I'll add links to components which have READMEs and add issues for those that do not